### DISCUSSION TITLE
[General] Is there no node guide for non-developers and regular users??

### DISCUSSION THREAD

**Discussion by Narzas** [zkSync-Community-Hub/zksync-developers#551] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Is there a light version node that can work with low specifications??
And is there no node quick installation guide for non-developers? 
If not, I would like to make a guide, is there any guide I can refer to??

**Comment by bxpana** [comment, marked as answer]
We have documentation at https://docs.zksync.io/infra

Is there something you're looking for specifically?
